### PR TITLE
[Doc] Shared data HDFS storage volume update

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ displayed_sidebar: "Chinese"
 
 > **注意**
 >
-> 仅拥有 SYSTEM 级 CREATE STORAGE VOLUME 权限的用户可以执行该操作。
+> - 仅拥有 SYSTEM 级 CREATE STORAGE VOLUME 权限的用户可以执行该操作。
+> - 如果您需要基于 HDFS 创建存储卷，建议您不要修改 **HADOOP_CONF** 和 **core-site.xml/hdfs-site.xml**，否则可能导致系统发生未知行为。
```
Comment (yingtingdong) [inline docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md:16]: 我理解这里是指conf文件和storage volume参数同时存在且不一致的话，可能会有未知的行为。和修改配置文件应该没啥关系。

Comment (EsoragotoSpirit) [reply docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md]: fixed

File: docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md [hunk 2]
```diff
@@ -252,64 +253,100 @@ PROPERTIES
...
+    | hadoop.security.authentication                        | 指定认证方式。有效值：`simple`（默认） 和 `kerberos`。`simple` 表示简单认证，即 Username。`kerberos` 表示 Kerberos 认证。 |
     | username                                              | 用于访问 HDFS 集群中 NameNode 节点的用户名。                 |
-    | password                                              | 用于访问 HDFS 集群中 NameNode 节点的密码。                   |
-    | kerberos_principal                                    | 用于指定 Kerberos 的用户或服务 (Principal)。每个 Principal 在 HDFS 集群内唯一，由如下三部分组成：<ul><li>`username` 或 `servicename`：HDFS 集群中用户或服务的名称。</li><li>`instance`：HDFS 集群要认证的节点所在服务器的名称，用来保证用户或服务全局唯一。比如，HDFS 集群中有多个 DataNode 节点，各节点需要各自独立认证。</li><li>`realm`：域，必须全大写。</li></ul>示例：`nn/zelda1@ZELDA.COM`。 |
-    | kerberos_keytab                                       | 用于指定 Kerberos 的 Key Table（简称为“keytab”）文件的路径。 |
-    | kerberos_keytab_content                               | 用于指定 Kerberos 中 keytab 文件的内容经过 Base64 编码之后的内容。该参数跟 `kerberos_keytab` 参数二选一配置。 |
+    | hadoop.security.kerberos.ticket.cache.path            | 用于指定 Kerberos 的 Key Table（简称为“keytab”）文件的路径。 |
```
Comment (kevincai) [inline docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md:336]: a custom kerberos cache path. 这个不是keytab文件, 是kinit生成的ticket缓存文件路径.

File: docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md [hunk 3]
```diff
@@ -252,64 +253,100 @@ PROPERTIES
...
+    | dfs.ha.namenodes.<ha_cluster_name\>                   | 自定义 NameNode 的名称，多个名称以逗号 (,) 分隔，双引号内不允许出现空格。其中 `<ha_cluster_name>` 为 `dfs.nameservices` 中自定义的 HDFS 服务的名称。 |
     | dfs.namenode.rpc-address.<ha_cluster_name\>.<NameNode\> | 指定 NameNode 的 RPC 地址信息。 其中 `<NameNode>` 表示 `dfs.ha.namenodes.<ha_cluster_name>` 中自定义 NameNode 的名称。 |
     | dfs.client.failover.proxy.provider                    | 指定客户端连接的 NameNode 的提供者，默认为 `org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider`。 |
+    | fs.viewfs.mounttable.<ViewFS_cluster\>.link./<viewfs_path\> | 需要挂载的 ViewFS 集群路径，多个路径以逗号 (,) 分隔。其中 `<ViewFS_cluster>` 为 `LOCATIONS` 中自定义的 ViewFS 集群名。 |
+    | fs.alluxio.impl                                       | 设置为 `alluxio.hadoop.FileSystem`。                       |
+    | fs.AbstractFileSystem.alluxio.impl                    | 设置为 `alluxio.hadoop.AlluxioFileSystem`。                |
+
+<!--
+
+    | kerberos_keytab_content                               | 用于指定 Kerberos 中 keytab 文件的内容经过 Base64 编码之后的内容。该参数跟 `kerberos_keytab` 参数二选一配置。 |
```
Comment (kevincai) [inline docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md:347]: 这些不支持的选项不要列.

* kerberos_keytab_content
* password
* kerberos_principal

Comment (EsoragotoSpirit) [reply docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md:347]: 这里用tag隐藏了，页面不会显示。只是方便后期如果要的话可以加回来

File: docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md [hunk 4]
```diff
@@ -315,32 +315,17 @@ PROPERTIES
...
     | **属性**                                              | **描述**                                                     |
     | ----------------------------------------------------- | ------------------------------------------------------------ |
-    | enabled                                               | 是否启用当前存储卷。默认值：`false`。已禁用的存储卷无法被引用。 |
+    | enabled                                               | 是否启用当前存储卷。默认值：`false`。已禁用的存储卷无法被引用。       |
     | hadoop.security.authentication                        | 指定认证方式。有效值：`simple`（默认） 和 `kerberos`。`simple` 表示简单认证，即 Username。`kerberos` 表示 Kerberos 认证。 |
-    | username                                              | 用于访问 HDFS 集群中 NameNode 节点的用户名。                 |
-    | hadoop.security.kerberos.ticket.cache.path            | 用于指定 Kerberos 的 Key Table（简称为“keytab”）文件的路径。 |
-    | dfs.nameservices                                      | 自定义 HDFS 集群的名称。                                     |
+    | username                                              | 用于访问 HDFS 集群中 NameNode 节点的用户名。                      |
+    | hadoop.security.kerberos.ticket.cache.path            | 用于指定 Klint 生成的 Ticket Cache 文件的路径。                   |
```
Comment (kevincai) [inline docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md:323]: Klint -> Kinit

**Review by kevincai — APPROVED** [review]

File: docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md [hunk 5]
```diff
@@ -252,64 +253,85 @@ PROPERTIES
...
-    | username                                              | 用于访问 HDFS 集群中 NameNode 节点的用户名。                 |
+    | enabled                                               | 是否启用当前存储卷。默认值：`false`。已禁用的存储卷无法被引用。       |
+    | hadoop.security.authentication                        | 指定认证方式。有效值：`simple`（默认） 和 `kerberos`。`simple` 表示简单认证，即 Username。`kerberos` 表示 Kerberos 认证。 |
+    | username                                              | 用于访问 HDFS 集群中 NameNode 节点的用户名。                      |
+    | hadoop.security.kerberos.ticket.cache.path            | 用于指定 kinit 生成的 Ticket Cache 文件的路径。                   |
+    | dfs.nameservices                                      | 自定义 HDFS 集群的名称。                                        |
+    | dfs.ha.namenodes.`<ha_cluster_name\>`                   | 自定义 NameNode 的名称，多个名称以逗号 (,) 分隔，双引号内不允许出现空格。其中 `<ha_cluster_name>` 为 `dfs.nameservices` 中自定义的 HDFS 服务的名称。 |
+    | dfs.namenode.rpc-address.`<ha_cluster_name\>`.`<NameNode\>` | 指定 NameNode 的 RPC 地址信息。 其中 `<NameNode>` 表示 `dfs.ha.namenodes.<ha_cluster_name>` 中自定义 NameNode 的名称。 |
```
Comment (evelynzhaojie) [inline docs/zh/sql-reference/sql-statements/Administration/CREATE_STORAGE_VOLUME.md:326]: 自定义的

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**